Gruntfile.js

```diff
@@ -133,43 +133,36 @@ module.exports = function( grunt ) {
 
 					"requirejs/require.js": "requirejs/require.js",
 					"requirejs-domready/domReady.js": "requirejs-domready/domReady.js",
-					"requirejs-text/text.js": "requirejs-text/text.js",
+					"requirejs-text/text.js": "requirejs-text/text.js"
 				}
 			}
 		},
-		jshint: {
+		eslint: {
 			options: {
-				jshintrc: true
+
+				// See https://github.com/sindresorhus/grunt-eslint/issues/119
+				quiet: true,
+				configFile: ".eslintrc",
+				fix: true
```

Comment:
`configFile` & `fix` shouldn't be needed. Compare with the [jQuery equivalent](https://github.com/jquery/jquery/blob/6ced2639b567ab4a5089f38a8e90efedc12801bf/Gruntfile.js#L99-L114).